repo/README.md

```diff
@@ -32,46 +31,35 @@ Repo Implementations:
 
```

Comment:
In go, we use https://github.com/jbenet/go-datastore/ -- which is similar and alraedy has a bunch. 

i've been making these abstractions for a long time: https://github.com/datastore/datastore